src/SixLabors.Fonts/RendererOptions.cs

```diff
@@ -10,14 +12,15 @@ namespace SixLabors.Fonts
     /// </summary>
     public sealed class RendererOptions
     {
+        private static FontFamily[] emptyFamilyCollection = new FontFamily[0];
```

Comment:
`Array.Empty<FontFamily>()`

Reply:
yep much better totally forgot that one existed